go/httpapi-lambda/README.md

```diff
@@ -0,0 +1,14 @@
+# Welcome to your CDK Go project!
```

Comment:
please expand this readme to include information from other readmes, at least. 

Go is no longer in developer preview.